### ISSUE TITLE
Write scripts to set up development environments for Portable

### ISSUE THREAD

**Issue by tst-basilhuffman** [google/earthenterprise#575]
As a corollary to PR# 475. There should exist a script which sets up the build environment for Portable also.

**Comment by tst-ppenev** [comment]
@tst-basilhuffman: The Docker setup scripts already set up the environment for Portable Server.  (The GEE Docker image, actually, already builds and installs Portable Server.)

**Comment by tst-basilhuffman** [comment]
Yes, I should have been more clear: this is for Windows and perhaps Mac, if the decision is ever made to officially support.
